core/math/random.cpp

```diff
@@ -126,6 +227,7 @@ void Random::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("range", "from", "to"), &Random::range);
 	ClassDB::bind_method(D_METHOD("choice", "from"), &Random::choice);
+	ClassDB::bind_method(D_METHOD("choices", "from", "count", "weights"), &Random::choices, DEFVAL(1), DEFVAL(Variant()), DEFVAL(false));
```

Comment:
Missing bind argument:

```suggestion
	ClassDB::bind_method(D_METHOD("choices", "from", "count", "weights", "cumulative"), &Random::choices, DEFVAL(1), DEFVAL(Variant()), DEFVAL(false));
```